src/components/LoginSignUp/Login.js

```diff
@@ -103,7 +99,7 @@ const Login = ( ): Node => {
         autoCapitalize="none"
         keyboardType="email-address"
         selectionColor={colors.black}
-        onFocus={e => scrollToInput( findNodeHandle( e.target ) )}
+        onFocus={() => setExtraScrollHeight( 200 )}
```

Comment:
I don't love this hardcoding, but I'm failing to find a better alternative so it works for now.